### PR TITLE
actually use the lockscreen implementation selected from settings

### PR DESCRIPTION
**Pull request by uvera**
```code
static const char *slockcmd[] = {"sh","-c", "eval $(iconf lockscreen)", NULL};
```
This might not be ideal, so I'm open for suggestions

### CODE REVIEW

File: config.def.h [hunk 1]
```diff
@@ -151,7 +151,7 @@ static const char *instantrepeatcmd[] = {"instantrepeat", NULL};
 static const char *instantpacmancmd[] = {"instantpacman", NULL};
 static const char *instantsharecmd[] = {"instantshare", "snap", NULL};
 static const char *nautiluscmd[] = {".config/instantos/default/filemanager", NULL};
-static const char *slockcmd[] = {"ilock", NULL};
+static const char *slockcmd[] = {"sh","-c", "eval $(iconf lockscreen)", NULL};
```
Comment (con-f-use) [inline config.def.h:154]:
```suggestion
static const char *slockcmd[] = {".config/instantos/default/lockscreen", NULL};
```

Comment (uvera) [reply config.def.h:154]: I cannot get to work it this way, settings app doesn't even write to that file nor does it want to execute from the WM keybind

Comment (con-f-use) [reply config.def.h]: That is a separate issue then. It should use this syntax, see the other uses of `~/.config/instantos/devault/` in `config.def.h`.

Comment (uvera) [reply config.def.h]: actually my bad, my `defaultapps` script wasn't up to date from the instantOS repo, it actually works now

Comment (uvera) [reply config.def.h]: The settings app actually works now, I guess it could be merged

**Pull request merged**